### DISCUSSION TITLE
Aggregating regions

### DISCUSSION THREAD

**Discussion by pbubik** [electricitymaps/electricitymaps-contrib#3309] [Ideas] [answered]
Can we make it possible to aggregate various regions?

Sometimes, it may be useful to know the average use use for a whole country. Like for example Russia, Canada, Australia or United States. Other times it may be useful to aggregate entire regions: like Europe, or South America...

Would it be possible to add a filter on the intro page that allows a user to aggregate regions?

Thank you.

**Comment by sandos** [comment]
I would love this, especially for Norway in my mind, and Sweden when it gets split up. Lots of detail is nice but overall summaries for countries are also very useful.

**Comment by madsnedergaard** [comment]
We'd love to do this and it's on our backlog(!), but it requires a significant change to our data model and engine, which is why it hasn't been prioritised yet. 

We'll try to keep this discussion up to date when there's any news :)


I tried merging this with an existing issue, but Github is annoying - so here's the content from https://github.com/tmrowco/electricitymap-contrib/issues/2954 to keep it together:

![Screenshot 2021-11-05 at 11 49 52](https://user-images.githubusercontent.com/3296643/140499503-fe504999-c09a-40d4-a804-75d896538eac.png)

**Comment by madsnedergaard** [comment, marked as answer]
Hi, this feature has now been done! 🥳 

We're slowly rolling it out for new regions and countries - as of this comment it's available for USA, Australia, Sweden, Norway and Italy.

**Reply by VIKTORVAV99** [reply]
And Brazil, Denmark and the Faroe Islands 😉
